test/distributions/test_distributions.py

```diff
@@ -342,6 +343,16 @@ def _get_examples():
                 },
             ],
         ),
+        Example(
```

Comment:
Is this new example the one that fails for the Weibull distribution?

Reply:
i dont believe so... the failing example is in the variance test for the Weibull due to tiny difference in tolerances, so perhaps due to a different state with the addition of a new distribution?